app/views/recipes/show.html.erb

```diff
@@ -39,21 +32,19 @@
       <% if @recipe.recipes_foods.any? %>
          <thead>
              <tr>
-             <th>No.</th>
              <th>Food</th>
              <th>Quantity</th>
              <th>Value</th>
              <th>Action</th>
              </tr>
          </thead>
-
+     
          <tbody>
-         <% @recipe.recipes_foods.each_with_index do |fd, i|%>
+         <% @recipe.recipes_foods.each do |fd|%>
          <tr>
-           <td><%= i+1%></th>
            <td><%= fd.food.name%></th>
            <td><%= fd.quantity%> <%= fd.food.measurement_unit  %></td>
```

Comment:
here it is showing in our page an extra digit '1' please adjust this code as I have said in other comment